pkg/virt-controller/watch/application_test.go

```diff
@@ -42,6 +43,7 @@ import (
 	k8sv1 "k8s.io/api/core/v1"
 	kubev1 "k8s.io/api/core/v1"
 	"k8s.io/api/policy/v1beta1"
+	discoveryFake "k8s.io/client-go/discovery/fake"
```

Comment:
A minor one - I find the came case usage here a little bit confusing. I almost never saw us using it for aliasing. I saw either a snake case or just in small letters.

Reply:
You're right, I actually didn't notice this as it probably got added by running `make`. Will fix, thanks!